cadc-log/src/main/java/ca/nrc/cadc/log/LogControlServlet.java

```diff
@@ -366,6 +370,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
         // redirect the caller to the resulting settings
         response.setStatus(HttpServletResponse.SC_SEE_OTHER);
         String url = request.getRequestURI();
+
+        // We could condense this and just redirect to one of them, but it seems more proper to just carry forward what
+        // the caller submitted.
+        String secretQuery = request.getParameter(SECRET_PROPERTY);
+        String secretHeader = request.getHeader(REQUEST_HEADER_SECRET_KEY);
+
+        if (StringUtil.hasLength(secretQuery)) {
+            url = url + "?" + SECRET_PROPERTY + "=" + secretQuery;
+        }
+
+        if (StringUtil.hasLength(secretHeader)) {
```

Comment:
I'm OK with the secret param so that `curl --location` works as expected.

The authorization should definitely *not* be set in the response.(it is a request header only).

Reply:
That was a dumb thing to do!  Fixed now.